watch.go

```diff
@@ -90,18 +92,67 @@ func liveEvents(lr Reloader, ch chan []string) {
 	}
 }
 
+func shouldInclude(file string, excludePatterns []string) bool {
+	for _, pattern := range excludePatterns {
+		match, err := filepath.Match(pattern, file)
+
+		if err != nil {
+			fmt.Println("Bad pattern: ", pattern)
```

Comment:
I tried the second approach, however `filepath.Match` won't always report an error on the empty string (because it doesn't have to read the whole pattern), so that doesn't work.